lib/rules/no-unescaped-entities.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @fileoverview HTML special characters should be escaped.
+ * @author Patrick Hayes
+ */
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+var DEFAULTS = ['>', '<', '"', '\'', '}'];
```

Comment:
I see the value in linting for `>`, since you have provided such a good example of what that can be problematic. However, I'm not so sure about the other characters listed in the defaults here. Can you provide similar reasoning for each of these characters?